### PR TITLE
profiles: use curl 7.41 or greater to pull in ipv4/ipv6 connection fixes

### PR DESCRIPTION
**Pull request by mischief**
updates coreos/bugs#324

### CODE REVIEW

**Comment by mischief** [comment]
depends on https://github.com/coreos/portage-stable/pull/183

File: profiles/coreos/base/package.accept_keywords [hunk 1]
```diff
@@ -111,3 +111,7 @@ dev-util/checkbashisms
 
 # Fixes findmnt with overlay and btrfs filesystems
 =sys-apps/util-linux-2.26.1
+
+# fix for https://github.com/coreos/bugs/issues/324
+>=net-misc/curl-7.41
```
Comment (marineam) [inline profiles/coreos/base/package.accept_keywords]: Just say =net-misc/curl-7.41 so we don't always run with ~amd64 once portage-stable updates to newer releases.

**Comment by mischief** [comment]
done. PTAL

**Comment by marineam** [comment]
LGTM

**Pull request merged**